examples/sample/src/main/java/com/cryptlex/sample/Sample.java

```diff
@@ -18,6 +20,8 @@ public static void main(String[] args) {
             LexActivator.SetProductData("PASTE_CONTENT_OF_PRODUCT.DAT_FILE");
             LexActivator.SetProductId("PASTE_PRODUCT_ID", LexActivator.LA_USER);
             LexActivator.SetReleaseVersion("1.0.0");  // Set this to the release version of your app
+            LexActivator.SetReleaseChannel("RELEASE_CHANNEL"); // Set this to the release channel of your app
```

Comment:
remove the release channel and platform functions

Reply:
Removed.